man/man5/zfs-events.5

```diff
@@ -804,7 +818,12 @@ Checksum for each range the number of bits cleared.
 \fBbad_set_bits\fR
 .ad
 .RS 12n
-Checksum array of bits set.
+If this field exists, it is an array of: (bad data & ~(good data)); that is,
+the bits set in the bad data which are cleared in the good data. Each element
+corresponds a byte whose offset is in a range in bad_ranges, and the array is
+ordered by offset. Thus the first element is the first byte in the first
+bad_ranges range and the last element is the last byte in the last bad_ranges
```

Comment:
`\fB` `\fR` around `bad_ranges`.
I'd suggests two commas:
  "Thus, the first...",
  "...range, and the last element..."

Reply:
done